File: Atributar/src/atributar/Atributar.java
/*
 * ESTE ES MI PRIMER PROGRAMA DE FPOO
DONDE VOY A CREAR OBJETOS (INSTANCIAR)
Y ADEMAS VOY A CREAR ATRIBUTOS

 */
package atributar;

/**
 * 2 abril 2018
 * @author Manuela Garcia Maonsalve
 */
public class Atributar {

   
    /**
     * atributos de tipos de carro: ejemplo
     * camionetas, camiones
     */
    
    String marca= "fOTON", modelo= "2017", color= "NEGRO", tamaño=" MEDIANO";
    long precio= 150000000; // atributos de los tipos de carro "estos pertenecen a camiones "
    public static void main(String[] args) { // clase main 
         
        Atributar camiones= new Atributar(); // se crea el primer objeto llamado camiones 
         
        
        System.out.println(" ----------------------CAMION----------------------");
       
        System.out.println(" la marca del camión es:" +camiones.marca); // se imprime la marca del camión
        System.out.println(" el modelo del camión:" +camiones.modelo); // se imprime el modelo del camión
        System.out.println(" el color del camión es:" +camiones.color); // se imprime el color del camión
        System.out.println(" el tamaño del camión es:" +camiones.tamaño); // se imprime el tamaño del camión 
        System.out.println(" el precio del camión es:" +camiones.precio); // se imprime el precio del camión
        System.out.println(" \n"); // se imprime un salto de linea 
        
              
         Atributar campero= new Atributar(); // se crea el segundo obejto llamado campero 
   
         
        
        campero.marca="toyota"; // se crea un nuevo valor al atributo marca para el objeto campero 
        campero.modelo="2016"; // se crea un nuevo valor al atributo modelo para el objeto campero 
        campero.color="rojo";  // se crea un nuevo valor al atributo color  para el objeto campero 
        campero.tamaño="pequeño"; // se crea un nuevo valor al atributo tamaño para el objeto campero 
        campero.precio=150000000; // se crea un nuevo valor al atributo precio para el objeto campero 
        System.out.println("  ----------------------CAMPERO---------------------- ");
        System.out.println(" la marca del campero es:" +campero.marca); // se imprime la marca del campero
        System.out.println(" el modelo del campero es:" +campero.modelo); // se imprime el modelo del campero
        System.out.println(" el color del campero es:" +campero.color); // se imprime el color del campero
        System.out.println(" el tamaño del campero es:" +campero.tamaño); // se imprime el tamaño del campero
        System.out.println(" el precio del campero es:" +campero.precio); // se imprime el precio del campero
        System.out.println(" \n"); // se realiza un salto de linea 
        
        Atributar automovil= new Atributar(); // se crea un tercer obejto llamado automovil
   
        
        
        automovil.marca="nissan"; // se crea un nuevo valor al atributo marca para el objeto automovil 
        automovil.modelo="2018"; // se crea un nuevo valor al atributo modelo para el objeto automovil 
        automovil.color="azul"; // se crea un nuevo valor al atributo color para el objeto automovil 
        automovil.tamaño="pequeño"; // se crea un nuevo valor al atributo tamaño para el objeto automovil 
        automovil.precio=70000000; // se crea un nuevo valor al atributo precio  para el objeto automovil 
        System.out.println("  ----------------------AUTOMOVIL----------------------");
        System.out.println(" la marca del automovil es:" +automovil.marca); // se imprime la marca del automovil
        System.out.println(" el modelo dle automovil es:" +automovil.modelo); // se imprime el modelo del automovil
        System.out.println(" el color del automovil es:" +automovil.color); // se imprime el color del automovil
        System.out.println(" el tamaño del automovil es:" +automovil.tamaño); // se imprime el tamaño del automovil
        System.out.println(" el precio del automovil es:" +automovil.precio); // se imprime el precio del automovil
         System.out.println(" \n"); // se realiza un salto de linea 
         
         
        Atributar clasico= new Atributar();
   
        
        
        clasico.marca="renault"; // se crea un nuevo valor al atributo marca para el objeto clasico
        clasico.modelo="2017"; // se crea un nuevo valor al atributo modelo para el objeto clasico
        clasico.color="blanco"; // se crea un nuevo valor al atributo color para el objeto clasico
        clasico.tamaño="mediano"; // se crea un nuevo valor al atributo tamaño para el objeto clasico
        clasico.precio=95000000; // se crea un nuevo valor al atributo  precio para el objeto clasico
        System.out.println("  ----------------------AUTO CLASICO ---------------------- ");
        System.out.println(" la marca del auto clasico es:" +clasico.marca); // se imprime la marca del auto clasico
        System.out.println(" el modelo del auto clasico es:" +clasico.modelo); // se imprime el modelo del auto clasico
        System.out.println(" el color del auto clasico es:" +clasico.color); // se imprime el color del auto clasico 
        System.out.println(" el tamaño del auto clasico es:" +clasico.tamaño); // se imprime el tamaño del auto clasico
        System.out.println(" el precio del auto clasico  es:" +clasico.precio); // se imprime el precio del auto clasico 
        
        System.out.println(" \n"); // se realiza un salto de linea 
        System.out.println(" EL PROGRAMA HA FINALIZADO"); // impresión por consola 

    }
    
}

File: atributar.py
class atributar: # se crea una clase llamada atributar

    #ESTE ES MI PRIMER PROGRAMA DE FPOO
    #DONDE VOY A CREAR OBJETOS (INSTANCIAR)
    #Y ADEMAS VOY A CREAR ATRIBUTOS




    #2 abril 2018
   # @author Manuela Garcia Maonsalve



    def __init__(self, marca: object, modelo: object, color: object, tamaño: object, precio: object) -> object:
        self.marca = marca   # se definen los atributos que poseeran cada uno de los objetos relacionados a los tipos de carros
        self.modelo = modelo
        self.color= color
        self.tamaño = tamaño
        self.precio= precio



camion= atributar('foton', 2017, 'rojo','grande',140000000) # se crea el primer obejto llamado camion
                                                            # a este objeto se le inicializan los atributos ya creados para la clase
print("----------------------------CAMIÓN--------------------------------")
print("la marca del camion es: " + camion.marca)  # se imprime la marca del camion
print("el modelo del camion es: " , camion.modelo)  # se imprime el modelo del camion
print ("el color del  camion es: ",  camion.color)   # se imprime el color del camion
print("el tamaño del camion es: ", camion.tamaño)    # se imprime el tamaño del camion
print("el precio del camion es: " , camion.precio)    # se imprime el precio del camion
print("\n")



campero= atributar('toyota', 2016, 'negro','pequeño',150000000)  # se crea el segundo objeto llamado campero
                                                                 # a este objeto se le inicializan los atributos ya creados para la clase
print("----------------------------CAMPERO--------------------------------")
print("la marca del campero es: " + campero.marca)  # se imprime la marca del campero
print("el modelo del campero es: " , campero.modelo)  # se imprime el modelo del campero
print ("el color del  campero es: ",  campero.color)   #  se imprime el color del campero
print("el tamaño del campero es: ", campero.tamaño)    # se imprime el tamaño del campero
print("el precio del campero es: " , campero.precio)   # # se imprime el precio del campero
print("\n")


automovil= atributar('nissan', 2018, 'blanco','pequeño',99000000)   # se crea el tercer objeto llamado automovil
                                                                # a este objeto se le inicializan  los atributos ya creados poara la clase
print("----------------------------AUTOMOVIL--------------------------------")
print("la marca del automovil es: " + automovil.marca)    # se imprime la marca del automovil
print("el modelo del automovil es: " , automovil.modelo)    # se imprime el modelo del automovil
print ("el color del  automovil es: ",  automovil.color)      #  se imprime el color del automovil
print("el tamaño del automovil es: ", automovil.tamaño)       # se imprime el tamaño del automovil
print("el precio del automovil es: " , automovil.precio)      #  # se imprime el precio del automovil
print("\n")


clasico= atributar('renault', 2015, 'gris','pequeño',80000000)   # se crea el cuarto objeto llamado clasico
                                                                # a este objeto se le inicializan los atributos ya creados para la clase
print("----------------------------AUTO CLASICO--------------------------------")
print("la marca del auto clasico es: " + clasico.marca)     # se imprime la marca del auto clasico
print("el modelo del auto clasico es: " , clasico.modelo)      # se imprime el modelo del auto clasico
print ("el color del auto clasico es: ",  clasico.color)       #  se imprime el color del auto clasico
print("el tamaño del auto clasico es: ", clasico.tamaño)        # se imprime el tamaño del auto clasico
print("el precio del auto clasico es: " , clasico.precio)     # se imprime el precio del auto clasico
print("\n")

